app/schema/user.py

```diff
@@ -14,8 +15,22 @@ class Role(enum.Enum):
     user = 2
 
 
-class User(Base):
+class User(Model):
     __tablename__ = "users"
 
-    id: UUID = Column(UUID(as_uuid=True), primary_key=True, index=True, default=uuid.uuid4)
```

Comment:
Should the typing be removed here? Tassilo ask to keep it above in the schema to register a user, why not here too?